pkg/cover/backend/elf.go

```diff
@@ -463,8 +463,16 @@ func readCoverPoints(target *targets.Target, file *elf.File, info *symbolInfo) (
 	return pcs, nil
 }
 
-func readModuleCoverPoints(file *elf.File, info *symbolInfo, mod *Module) ([2][]uint64, error) {
+func readModuleCoverPoints(target *targets.Target, file *elf.File, info *symbolInfo, mod *Module) ([2][]uint64, error) {
 	var pcs [2][]uint64
+	type Arch struct {
+			opcodeOffset int
+	}
+	arch := map[string]Arch{
+			targets.ARM64: {
+					opcodeOffset: 3,
```

Comment:
We already have this info in readCoverPoints functions. I think it's better to reuse that instead of duplicating. Consider, somebody will add support for another arch readCoverPoints, but will highly likely miss this duplicated info in readModuleCoverPoints. If we reuse readCoverPoints info, any support for other arches in readCoverPoints will automatically make modules work for that arch as well.
I think we need to move Arch type/map from readCoverPoints into a global var (rename somehow to make less ambiguous) and reuse here.